### ISSUE TITLE
Action Check for document_settings

### ISSUE THREAD

**Issue by meceware** [elementor/elementor#15588]
Hi,

I'm an Elementor plugin developer and I have an issue with the `document_settings` hook. On my plugin, I use 

```
elementor/element/wp-post/document_settings/after_section_end
elementor/element/wp-page/document_settings/after_section_end
```

hooks, which worked until now. On one of my clients, these hooks are changed somehow and called as `post` instead of `wp-post` and `page` instead of `wp-page`. At first I thought another plugin is interfering with it or the client modified Elementor plugin but these weren't the case. 

Can anyone know why these hooks are changed and under what circumstances can these hooks change? Maybe there is an option that might change the behaviour of these hooks.

Thank you,

**Comment by ramiy** [comment]
@meceware Can you provide more information? What Elementor version broke your code?

Please note that I've checked [this hooks](https://github.com/elementor/elementor/blob/2cd3d50d8d87edde9cd3c820f5a0f8b1f2c4f24f/includes/base/controls-stack.php#L1450-L1462) and the last time it was changed was over 3 years ago. Sounds like it's not the hook, it's the document name (the `{$stack_name}` variable).

**Comment by meceware** [comment]
You are right, that's the `$stack_name` variable that comes from the child class itself that calls the hook.

The problem I'm seeing is on Elementor `3.2.4`, I reinstalled the Elementor and didn't change anything. 

The code that doesn't work on my client's server is

```
add_action( 'elementor/element/wp-post/document_settings/after_section_end', array( $this, 'OnElementorAfterSectionEnd' ), 10, 2 );
add_action( 'elementor/element/wp-page/document_settings/after_section_end', array( $this, 'OnElementorAfterSectionEnd' ), 10, 2 );
```

and it works as

```
add_action( 'elementor/element/post/document_settings/after_section_end', array( $this, 'OnElementorAfterSectionEnd' ), 10, 2 );
add_action( 'elementor/element/page/document_settings/after_section_end', array( $this, 'OnElementorAfterSectionEnd' ), 10, 2 );
```

but this way doesn't work on other WordPress sites. I tried to print the hooks from the Elementor in `end_controls_section` method of `control-stack.php`, the only different is these hooks. The `$stack_name` variable is different on their server.
